spec.html

```diff
@@ -5216,23 +5216,21 @@ <h1>
       </h1>
       <dl class="header">
         <dt>description</dt>
-        <dd>It converts _argument_ to one of 2<sup>8</sup> integral Number values in the inclusive interval from *+0*<sub>𝔽</sub> to *255*<sub>𝔽</sub>.</dd>
+        <dd>It clamps _argument_ to one of 2<sup>8</sup> integral Number values in the inclusive interval from *+0*<sub>𝔽</sub> to *255*<sub>𝔽</sub>.</dd>
```

Comment:
I don't love this change. Clamping is only one of the things it does. There was nothing wrong with saying "converts" here. It's a mapping, and not one that can be otherwise simply explained IMO.

Reply:
This was to align with [ToLength](https://tc39.es/ecma262/multipage/abstract-operations.html#sec-tolength), which also uses "clamps" despite the inclusion of rounding. Should we instead change that one?

Reply:
I don't care much about the wording and mildly prefer consistency, so I'm happy with the change in this PR.

I'd also be fine with changing both to  be "clamps and rounds", to be precise.

Reply:
> I'd also be fine with changing both to be "clamps and rounds", to be precise.

Ooh, I like that more.

Reply:
Me too. Done, in both operations.